### DISCUSSION TITLE
Pessoal, qual a opinião de você sobre os dois modelos de Notebook: Samsung Galaxy book2-core i7 e Asus vivo book 16- core i7?

### DISCUSSION THREAD

**Discussion by Vini1684** [frontendbr/forum#2420] [Perguntas e Respostas]
Pessoal! Estou em dúvida de qual note comprar e gostaria da opinião de vocês para ajudar a decidir. Tive um Asus s46c que por sinal foi muito bom e durou horrores (+1 década). No entanto, ele parou e resolvi comprar outro.

A ideia é gastar até 4k. O uso será para edicao de vídeos, alguns jogos leves esporadicamente, alguns programas como AutoCAD 3d, Power bi; assistir aulas, !

Fique na dúvida entre dois modelos:
1-asus vivo book 16( core i7- 1255u, HD 257 GB e RAM de 8gb) - preço: 3.279,00
2- Samsung Galaxy book2 ( core i7-1255u, HD 512 GB e RAM de 8 Gb - preço: 3.779,00)
Ambos possuem placa gráfica Intel ires xe. Já ouvi falar que a Samsung não é bem conceituada quanto a Asus, mas em alguns reviews ouvi falar muito bem desse modelo citado acima da Samsung. Qual a opinião de vocês sobre ? Podem me ajudar a decidir por gentileza! Gratidão 🙏😊

**Comment by victorinknov** [comment]
Nenhum dos dois. Compre este e seja feliz:

https://www.lenovo.com/br/pt/laptops/ideapad/serie-300/IdeaPad-3-15ALC6/p/82MFS00600?cid=br:sem|se|google|j-b2c-amd-convers-google-performancemax-advantage|||pt_BR82MFS00600|17923677094|||pmax|mixed|all&gclid=Cj0KCQjw_O2lBhCFARIsAB0E8B9TTiRwRjccudoH4ODfoowMmbVQeReI709GmePq_SiCdjHocnFwDG4aAmHWEALw_wcB

https://www.mercadolivre.com.br/notebook-ultrafino-ideapad-3-r7-12gb-512gb-linux-82mfs00600/p/MLB23998565

**Comment by victorinknov** [comment]
edit: o notebook que mandei acima estava por R$3.059 e compensava muito. agora, depois dessa thread fui buscar novos modelos e achei esse por R$4.079, se seu orçamento é até 4k por um note, essa é a melhor opção de produto novo (primeira mão). tem 8gb 256 ssd m2 e uma gtx 1650.

https://www.lenovo.com/br/pt/laptops/ideapad/ideapad-gaming-laptops/Gaming-3-Gen-6-15-AMD/p/WMD00000479

**Reply by Vini1684** [reply]
Curti! Tava dando uma olhada e vi que ele tem um AMD Ryzen 7 - 5800. Vi uns comparativos e ele se compara de frente ao I7 1165, geração passada de processadores e que perde, na pontuação  geral, para o i7 1255u, apesar de ganhar em performance e Benchmark : https://versus.com/br/amd-ryzen-7-5800-vs-intel-core-i7-1255u#group_performance

Haveria.muita diferença, considerando que a versão indicado do Lenovo tem placa gráfica?! Realmente sou o leigo buscando conhecimento com base em info e vcs para tomar a decisão mais assertiva. Obrigado!

**Reply by victorinknov** [reply]
Diferença absurda, sem contar que com a placa gráfica vai conseguir rodar alguns games caso queira.
